Backend/server.js

```diff
@@ -2,12 +2,14 @@ require('dotenv').config({path: __dirname + '/.env'})
 const express = require('express');
 const cors = require('cors');
 const mongoose = require('mongoose');
+const {scraper} = require('./scraper/primeScraper');
```

Comment:
I'm curious why this works. I never call the scraper function in the rest of the code (see line 24) but it runs it

Reply:
That's weird. We shouldn't have it run unless called. It'll be easier to figure out once all our code is merged tho